pkg/client/clientset.go

```diff
@@ -25,9 +25,9 @@ import (
 
 	"golang.org/x/oauth2/clientcredentials"
 
-	coreapi "github.com/dynatrace/dynatrace-configuration-as-code-core/api"
-	automationApi "github.com/dynatrace/dynatrace-configuration-as-code-core/api/clients/automation"
-	corerest "github.com/dynatrace/dynatrace-configuration-as-code-core/api/rest"
+	libAPI "github.com/dynatrace/dynatrace-configuration-as-code-core/api"
```

Comment:
We agreed to split refactorings(name changes) into separate small PRs to make it easier to review features.

Reply:
Moved to https://github.com/Dynatrace/dynatrace-configuration-as-code/pull/1723

Reply:
moved to https://github.com/Dynatrace/dynatrace-configuration-as-code/pull/1724